v0.1/reference_submissions/person_detection/parse_coco.py

```diff
@@ -0,0 +1,174 @@
+# Lint as: python3
+'''Extract the MS COCO dataset into person/not person training dataset.
+
+Extract and segment the MS COCO dataset into person and no person image sets of
+equal size. Save images in dataset/person and dataset/no_person subsets ready
+for training.
+
+The 2017 train images and train/val annotations must be downloaded from
+https://cocodataset.org/#download and extracted before this script can be run.
+The images should be extracated in the same directory as this script.
+'''
+import glob
+import json
+import os
+
+from absl import app
+
+from PIL import Image
+from resizeimage import resizeimage
+
+IMAGE_DIMS = 128
+OUTPUT_DIR = 'dataset'
+
+
+def write_image(image_id, annotation_file_name, is_person):
+    image_name = "{}.jpg".format(str(image_id).zfill(12))
+    image_dir = annotation_file_name.split('_')[1].split('.')[0] + '/'
+    image_path = os.path.join(image_dir, image_name)
+
+    with Image.open(image_path) as image:
+        cover = resizeimage.resize_cover(image, [IMAGE_DIMS, IMAGE_DIMS])
+        if is_person:
+            save_file_name = "{}/person/{}".format(OUTPUT_DIR, image_name)
+        else:
+            save_file_name = "{}/no_person/{}".format(OUTPUT_DIR, image_name)
+        cover.save(save_file_name, image.format)
+
+def ensure_image_quality(annotation, image):
+    bounding_box = annotation['bbox']
+    if annotation['iscrowd'] != 0:
+        return False
+
+    width = image['width']
+    height = image['height']
+    min_dimension = min(width, height)
+
+    # Ensure person occupies 20% of image area
```

Comment:
I agree with this change but I think we don't want to deviate from the standard VWW dataset without some concrete reason.

Is this modification used for the TFLM example?